stackit/internal/services/iaas/network/resource.go

```diff
@@ -40,10 +41,19 @@ type Model struct {
 	NetworkId        types.String `tfsdk:"network_id"`
 	Name             types.String `tfsdk:"name"`
 	Nameservers      types.List   `tfsdk:"nameservers"`
+	IPv4Gateway      types.String `tfsdk:"ipv4_gateway"`
+	IPv4Nameservers  types.List   `tfsdk:"ipv4_nameservers"`
+	IPv4Prefix       types.String `tfsdk:"ipv4_prefix"`
 	IPv4PrefixLength types.Int64  `tfsdk:"ipv4_prefix_length"`
 	Prefixes         types.List   `tfsdk:"prefixes"`
```

Comment:
What is the difference between `Prefixes` and `IPv4Prefix`?

Reply:
Couldn't they be the same field?

Reply:
One is computed only and the other is optional

Reply:
In the API design, there are `two` seperate fields. 
Users can give a single `prefix` while creating a network. But in the response, there is a new field called `prefixes` where list of the prefixes are returned.

Reply:
But for uniformity shouldn't we call them `IPv4Previx` and `IPv4Prefixes`? Like we do for the IPv6 ones

Reply:
Instead of`IPv4Previx` and just `Prefixes`

Reply:
@joaopalet yes we can do this. In that case, I need to deprecate `Prefixes` (even though it is only computed) field and create a new field `IPv4Prefixes`.

Reply:
Deprecated the `Prefixes` field and created new field `IPv4Prefixes`